src/server/ssrServer.js

```diff
@@ -68,6 +68,11 @@ export async function createApp(opts = {}) {
   fastify.register(ensureCorrelationId);
   fastify.register(fastifyCookie);
   fastify.register(logging);
+  fastify.register(fastifyMetrics, {
+    defaultMetrics: { enabled: false },
+    endpoint: null,
```

Comment:
Why do we set endpoint to null? how exactly does this change how some one will retrieve the metrics itself? im assuming none but still wondering

Reply:
there is already a separate metrics server running. this disables the plugin from adding metrics routes to the main one app server